src/Cache/Engine/FileEngine.php

```diff
@@ -256,7 +256,12 @@ public function clear(): bool
                 continue;
             }
 
-            $path = $path->getRealPath() . DIRECTORY_SEPARATOR;
+            $realPath = $path->getRealPath();
```

Comment:
Here and above this line `$path` is `SplFileInfo` and `string` below.

Rename it in upper part to something like `$fileinfo`.

Reply:
done